vim_binding.js

```diff
@@ -406,8 +410,8 @@ define([
       'z,t': 'ipython.scroll-cell-top',
       'ctrl-j': 'ipython.select-next-cell',
       'ctrl-k': 'ipython.select-previous-cell',
-      'shift-j': 'ipython.extend-selection-next',
-      'shift-k': 'ipython.extend-selection-previous',
+      'shift-j': 'ipython.extend-selection-below',
```

Comment:
Note: It was incorrect https://github.com/jupyter/notebook/blob/4.x/notebook/static/notebook/js/actions.js#L420